OpenLIFUData/OpenLIFUData.py

```diff
@@ -804,6 +806,9 @@ def enter(self) -> None:
         # Make sure parameter node exists and observed
         self.initializeParameterNode()
 
+        # Temp code to test that the data module is aware of guided mode when entering module
+        print("Setting up in Guided Mode?:", get_guided_mode_status())
+
```

Comment:
remember to get rid of this temporary code (but ty i benefited from having it while testing)